apps/angular/src/core/services/form-error.service.ts

```diff
@@ -0,0 +1,107 @@
+
+import { inject, Injectable } from '@angular/core';
+import { AbstractControl, FormGroup } from '@angular/forms';
+import { ApiError } from '@js-camp/core/models/api-error';
+
+import { NotificationService } from './notification.service';
+
+/** Form error service. */
+@Injectable({
+	providedIn: 'root',
+})
+export class FormErrorService {
+	private readonly notificationService = inject(NotificationService);
+
+	// eslint-disable-next-line @typescript-eslint/naming-convention
+	private readonly ERROR_MESSAGES: {[key: string]: string;} = {
+		general: 'Something went wrong. Please try again later.',
+		required: 'This field is required.',
+		email: 'Enter a valid email address.',
+		minlength: `Below the minimum length required for this field.`,
+		maxlength: `Exceeds the maximum length of this field.`,
+		passwordMatch: 'Passwords do not match.',
+	};
```

Comment:
You can extract it to variable. Also prefer using `Record<>`, its a bit cleaner 
```suggestion
	const ERROR_MESSAGES: Record<string, string> = {
		general: 'Something went wrong. Please try again later.',
		required: 'This field is required.',
		email: 'Enter a valid email address.',
		minlength: `Below the minimum length required for this field.`,
		maxlength: `Exceeds the maximum length of this field.`,
		passwordMatch: 'Passwords do not match.',
	};
```